common/list/lists-mws-fr.html

```diff
@@ -0,0 +1,377 @@
+				<ul class="lst-none">
+					<li>Élément de liste 2a</li>
+					<li>Élément de liste 2b</li>
+				</ul>
+			</li>
+			<li>Élément de liste 2</li>
+			<li>Élément de liste 3</li>
+		</ul>
+		<h3>Code</h3>
+		<pre><code>// Liste non stylisée :
+&lt;ul <strong>class=&quot;list-unstyled&quot;</strong>&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ul&gt;
+// Liste imbriquée non stylisée :
+&lt;ul&gt;
+	&lt;li&gt;Élément de liste 1
+		&lt;ul <strong>class=&quot;lst-none&quot;</strong>&gt;
+			&lt;li&gt;Élément de liste 1a&lt;/li&gt;
+			&lt;li&gt;Élément de liste 1b&lt;/li&gt;
+		&lt;/ul&gt;
+	&lt;/li&gt;
+&lt;/ul&gt;
+</code></pre>
+	</div>
+</div>
+<div class="row">
+	<h2>Listes numérotées</h2>
+	<div class="col-md-6">
+		<h3>Balisage personnalisé</h3>
+		<ol>
+			<li>Élément de liste 1</li>
+			<li>
+				Élément de liste 2
+				<div class="lst-num">
+					<ol>
+						<li>Élément de liste 2a</li>
+						<li>Élément de liste 2b</li>
+					</ol>
+				</div>
+			</li>
+			<li>Élément de liste 3</li>
+		</ol>
+		<h3>Code</h3>
+		<pre><code>&lt;ol&gt;
+	&lt;li&gt;...
+		&lt;div <strong>class=&quot;lst-num&quot;</strong>&gt;
+			&lt;ol&gt;
+				&lt;li&gt;...&lt;/li&gt;
+			&lt;/ol&gt;
+		&lt;/div&gt;
+	&lt;/li&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+	</div>
+	<div class="col-md-6">
+		<h3>Balisage par défaut - <span class="label label-success mrgn-lft-sm">Recommandé</span></h3>
+		<ol>
+			<li>Élément de liste 1</li>
+			<li>
+				Élément de liste 2
+				<ol class="lst-num">
+					<li>Élément de liste 2a</li>
+					<li>Élément de liste 2b</li>
+				</ol>
+			</li>
+			<li>Élément de liste 3</li>
+		</ol>
+		<h3>Code</h3>
+		<pre><code>&lt;ol&gt;
+	&lt;li&gt;...
+		&lt;ol <strong>class=&quot;lst-num&quot;</strong>&gt;
+			&lt;li&gt;...&lt;/li&gt;
+		&lt;/ol&gt;
+	&lt;/li&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+	</div>
+</div>
+<div class="row">
+	<h2>Listes alphabétiques − Lettre minuscule</h2>
+	<div class="col-md-6">
+		<h3>Balisage personnalisé</h3>
+		<ol>
+			<li>Élément de liste 1</li>
+			<li>
+				Élément de liste 2
+				<div class="lst-lwr-alph">
+					<ol>
+						<li>Élément de liste 2a</li>
+						<li>Élément de liste 2b</li>
+					</ol>
+				</div>
+			</li>
+			<li>Élément de liste 3</li>
+		</ol>
+		<h3>Code</h3>
+		<pre><code>&lt;ol&gt;
+	&lt;li&gt;...
+		&lt;div <strong>class=&quot;lst-lwr-alph&quot;</strong>&gt;
+			&lt;ol&gt;
+				&lt;li&gt;...&lt;/li&gt;
+			&lt;/ol&gt;
+		&lt;/div&gt;
+	&lt;/li&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+	</div>
+	<div class="col-md-6">
+		<h3>Balisage par défaut - <span class="label label-success mrgn-lft-sm">Recommandé</span></h3>
+		<ol>
+			<li>Élément de liste 1</li>
+			<li>
+				Élément de liste 2
+				<ol class="lst-lwr-alph">
+					<li>Élément de liste 2a</li>
+					<li>Élément de liste 2b</li>
+				</ol>
+			</li>
+			<li>Élément de liste 3</li>
+		</ol>
+		<h3>Code</h3>
+		<pre><code>&lt;ol&gt;
+	&lt;li&gt;...
+		&lt;ol&gt;
```

Comment:
```suggestion
		&lt;ol class="lst-lwr-alph">
```

Reply:
Fixed.

Reply:
I don't see this change. 

The code sample should match the demoed working example. Here it's a ordered number list where it contains a ordered lower alpha list. You did the reverse. Thanks for fixing it.